llvm/lib/Support/APInt.cpp

```diff
@@ -33,13 +33,13 @@ using namespace llvm;
 
 /// A utility function for allocating memory, checking for allocation failures,
 /// and ensuring the contents are zeroed.
-inline static uint64_t* getClearedMemory(unsigned numWords) {
+inline static uint64_t *getClearedMemory(unsigned numWords) {
   return new uint64_t[numWords]();
 }
 
 /// A utility function for allocating memory and checking for allocation
 /// failure.  The content is not zeroed.
-inline static uint64_t* getMemory(unsigned numWords) {
```

Comment:
It looks like you've reformatted this file (and the tests). It's best only to format your changes, which can be done by disabling options such as "format on save" in your editor and using `git clang-format HEAD~1` format only your changes. :slightly_smiling_face: